templates/web/base/waste/bulky/choose_date.html

```diff
@@ -8,6 +8,15 @@
   [% IF property %]
     [% INCLUDE 'waste/_address_display.html' %]
   [% END %]
+  [% IF cobrand.moniker == 'sutton' %]
+    <p>The list displays the available collection dates for your address for the next eight weeks.
+      Collections take place any time from 6:00am to 4:30pm. You do not need to be in on the collection day, just
+      make sure the items are placed outside before 6:00am on the collection day.</p>
+  [% ELSIF cobrand.moniker == 'kingston' %]
+  <p>The list displays the available collection dates for your address for the next eight weeks.
+  Collections take place any time from 6:30am to 4:30pm. You do not need to be in on the collection day, just
+  make sure the items are placed outside before 6:30am on the collection day.</p>
```

Comment:
As with part 1, this time is in a cobrand function somewhere, so the text could be shared between kingston and sutton, and the time looked up.